npm/private/test/yaml_tests.bzl

```diff
@@ -509,7 +509,7 @@ def _parse_errors(ctx):
 
     asserts.equals(env, None, parse("""\
 asdljfk
-            sdlkfdslkjf
+            gisdlkfdslkjf
```

Comment:
is this edit load-bearing?

Reply:
Yes. I'm sorry. I thought I have rollbacked it.